packages/pyroscope-flamegraph/src/format/format.ts

```diff
@@ -14,6 +14,12 @@ export function ratioToPercent(ratio: number) {
   return Math.round(10000 * ratio) / 100;
 }
 
+export function percentDiff(leftPercent: number, rightPercent: number): number {
+  // difference between 2 percents
+  // https://en.wikipedia.org/wiki/Relative_change_and_difference
+  return ((rightPercent - leftPercent) / leftPercent) * 100;
+}
```

Comment:
Since we are at here, can you unify that with https://github.com/pyroscope-io/pyroscope/blob/c56a6970cfc576c9eb572f4e6d7130576ab9f926/packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphComponent/color.ts#L26